vpd-manager/manager.cpp

```diff
@@ -387,7 +387,7 @@ void Manager::writeKeyword(const sdbusplus::message::object_path path,
         if (!std::get<1>(frus.find(objPath)->second).empty())
         {
             EditorImpl edit(std::get<1>(frus.find(objPath)->second), jsonFile,
-                            recordName, keyword);
+                            recordName, keyword, objPath);
```

Comment:
Since we don't update cache for redundant eeprom, there is no use in sending objPath parameter. Also am not seeing error log that is logging objPath. Can you please point me the usecase of having this objPath arg here?

Reply:
right. sorry I didn't notice catching the exception there at impl.cpp.